src/zope/app/i18n/browser/tests/test_exportimport.py

```diff
@@ -0,0 +1,92 @@
+##############################################################################
+#
+# Copyright (c) 2017 Zope Foundation and Contributors.
+# All Rights Reserved.
+#
+# This software is subject to the provisions of the Zope Public License,
+# Version 2.1 (ZPL).  A copy of the ZPL should accompany this distribution.
+# THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND ALL EXPRESS OR IMPLIED
+# WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT, AND FITNESS
+# FOR A PARTICULAR PURPOSE.
+#
+##############################################################################
+"""This module tests the Gettext Export and Import funciotnality of the
+Translation Domain.
+
+"""
+import unittest
+import time
+from io import BytesIO
+
+from zope.component.testlayer import ZCMLFileLayer
+
+from zope.app.i18n.translationdomain import TranslationDomain
+from zope.app.i18n.browser.exportimport import ExportImport
+import zope.app.i18n.tests
+from zope.publisher.browser import TestRequest
+
+
+class TestExportImport(unittest.TestCase):
+
+    layer = ZCMLFileLayer(zope.app.i18n.tests)
+
+    data = b'''
+
+msgid ""
+msgstr ""
```

Comment:
I just copied that from `test_filters.py` and didn't make any changes there to avoid introducing extraneous whitespace differences. I'll see what I can do.